runners/spark/src/main/java/org/apache/beam/runners/spark/translation/TransformTranslator.java

```diff
@@ -579,6 +597,7 @@ public void evaluate(View.CreatePCollectionView<ReadT, WriteT> transform,
     EVALUATORS.put(View.AsIterable.class, viewAsIter());
     EVALUATORS.put(View.CreatePCollectionView.class, createPCollView());
     EVALUATORS.put(Window.Bound.class, window());
+    EVALUATORS.put(TestStorageLevelPTransform.class, testStorageLevel());
```

Comment:
Separate this from the others with a comment line ? "test evaluators" or something ?